lib/qesapdeployment.pm

```diff
@@ -697,6 +698,7 @@ sub qesap_ansible_cmd {
     foreach (qw(provider cmd)) { croak "Missing mandatory $_ argument" unless $args{$_}; }
     $args{user} ||= 'cloudadmin';
     $args{filter} ||= 'all';
+    $args{timeout} //= bmwqemu::scale_timeout(90);
```

Comment:
args was already there but not checked as mandatory with the `croak` and no default value was assigned to it